front-end/src/app/shared/models/transaction-types/LOAN_MADE.model.ts

```diff
@@ -0,0 +1,51 @@
+import { schema } from 'fecfile-validate/fecfile_validate_js/dist/LOAN_MADE';
+import { SchBTransaction, ScheduleBTransactionTypes, ScheduleBTransactionTypeLabels } from '../schb-transaction.model';
+import { TemplateMapKeyType } from '../transaction-type.model';
+import { SchBTransactionType } from '../schb-transaction-type.model';
+import { LabelUtils } from 'app/shared/utils/label.utils';
+import { COMMITTEE, CORE_FIELDS } from 'app/shared/utils/transaction-type-properties';
+
+export class LOAN_MADE extends SchBTransactionType {
+  override formFields = [...CORE_FIELDS, 'organization_name', 'committee_fec_id'];
```

Comment:
any reason you don't use ...COM_FIELDS from transaction-type-properties?

Reply:
Yes, because COM_FIELDS has the 'committee_name' field but the spec for LOAN_MADE does not contain the beneficiary_committee_name field, it just has the payee_organization_name